en/advanced_config/ethernet_setup.md

```diff
@@ -0,0 +1,85 @@
+# PX4 Ethernet Setup
+
+Ethernet connectivity a faster, more reliable, and more flexible communication technology than serial communication approach.
```

Comment:
Is this true? I mean 
-  is it really faster, more reliable and more flexible than a serial port? How much? 
- IF there is an Ethernet, is that what we recommend instead of USB for connecting QGC to PX4
- IF there is an Ethernet, is that what we recommend instead of USB or another serial cable for connecting a  companion to PX4?

I'm not doubting you - I just don't know. My guess is that it probably the main benefits are if you are connecting to IP radios and anything that is "natively IP".

Reply:
Yeah, I'm not sure without "evidence". I think the main use case is likely to be to connect to things like IP radios.

I've adjusted the comment to make it clear that it is a great alternative, but not to add the value judgement "you should use this if you can".